src/db/crud.py

```diff
@@ -0,0 +1,93 @@
+from typing import List
+from sqlalchemy.orm import Session
+
+from . import models, schema
+
```

Comment:
please add return types to the function signatures, they are very helpful

Reply:
python3 FTW